test.sh

```diff
@@ -16,6 +16,35 @@ function join_by {
     fi
 }
 
+function uninstall_helm_repo() {
+    repo_name=$1
+    # clusterrole being pesky here so just uninstall in all namespaces
+    # TODO: find a way to only install non-cluster level resources
+    if helm list -a -A | grep -q "$repo_name"; then
+        namespaces=$(helm list -a -A --filter "$repo_name" | awk 'NR>1 {print $2}' | uniq)
+        for namespace in $namespaces
+        do
+            echo 1>&2 "Uninstalling $repo_name in $namespace"
+            helm uninstall "$repo_name" -n "$namespace"
+        done
+    fi
+}
+
+function add_helm_repo() {
+    repo_name=$1
+    repo_url=$2
+    echo 1>&2 "Adding Helm Repo $repo_name"
+    helm repo add "$repo_name" "$repo_url"
+}
+
+function install_helm_repo() {
+    repo_name=$1
+    chart=$2
+    version=$3
+    echo 1>&2 "Installing $repo_name"
+    helm install "$repo_name" "$chart" -n "$NAMESPACE" --version "$version"
```

Comment:
Maybe using `helm upgrade --install` instead would allow running this without checking if it's already installed.